docs/deployment-aws.md

```diff
@@ -0,0 +1,200 @@
+# DCL Deployment on AWS using Terraform and Ansible
+## Overview
+This document describes all necessary steps to deploy a new DCL network on AWS cloud in accordance with this [design document](./deployment-design-aws.md).
```

Comment:
The super-linter complains when headers doesn't contain a blank line before and after
```suggestion
# DCL Deployment on AWS using Terraform and Ansible

## Overview

This document describes all necessary steps to deploy a new DCL network on AWS cloud in accordance with this [design document](./deployment-design-aws.md).
```